lib/handler.js

```diff
@@ -76,6 +81,7 @@ Handler.prototype.checkRequestAuthorization = async function checkRequestAuthori
       await this.saveData({ accessToken: existingToken })
     }
   } catch (e) {
+    errorLog("nuxt-oauth - saveData failed, clear the session")
```

Comment:
Would logging the error also be helpful here?

Reply:
yeah I think @naveen-soho we need to make sure these errorLogs are actually error situations. so they should be instances where we're throwing or catching errors and also making sure the errors are logged too

so for example if they don't have an accessToken, is that a situation that should never occur? incase we need an errorLog. If it could reasonably occur normally we don't need to log

we want to make sure abnormal situations are logged and captured in raygun etc